alignak/modulesmanager.py

```diff
@@ -143,58 +121,85 @@ def load(self, modules):
         """
         self.modules_assoc = []
         for module in modules:
-            logger.info("Importing Python module '%s' for %s",
+            logger.info("Importing Python module '%s' for %s...",
                         module.python_name, module.module_alias)
             try:
                 python_module = importlib.import_module(module.python_name)
-                self.find_module_properties_and_get_instance(python_module, module.python_name)
+
+                # Check existing module properties
+                # Todo: check all mandatory properties
+                if not hasattr(python_module, 'properties'):
+                    self.configuration_errors.append(
+                        "Module %s is missing a 'properties' dictionary" % module.python_name
+                    )
+                    raise AttributeError
+                logger.info("Module properties: %s", getattr(python_module, 'properties'))
+
+                # Check existing module get_instance method
+                if not hasattr(python_module, 'get_instance') or \
+                        not callable(getattr(python_module, 'get_instance')):
+                    self.configuration_errors.append(
+                        "Module %s is missing a 'get_instance' function" % module.python_name
+                    )
+                    raise AttributeError
+
                 self.modules_assoc.append((module, python_module))
+                logger.info("Imported '%s' for %s", module.python_name, module.module_alias)
             except ImportError as exp:
-                logger.warning("Module %s (%s) can't be loaded, Python importation error",
-                               module.python_name, module.module_alias)
-                logger.exception("Exception: %s", exp)
-            except AttributeError as exp:
-                logger.warning("Module %s (%s) can't be loaded because attributes errors",
-                               module.python_name, module.module_alias)
-                logger.exception("Exception: %s", exp)
+                self.configuration_errors.append(
+                    "Module %s (%s) can't be loaded, Python importation error: %s" %
+                    (module.python_name, module.module_alias, str(exp))
+                )
+            except AttributeError:
+                self.configuration_errors.append(
+                    "Module %s (%s) can't be loaded, module configuration" %
```

Comment:
Perhaps add more info in log like _properties and/or get_instance_ missing to help the user / developer